toto/runlib.py

```diff
@@ -100,6 +100,11 @@ def record_artifacts_as_dict(artifacts):
     return artifacts_dict
 
   for artifact in artifacts:
+
+    if not os.path.exists(artifact):
+      log.warning("path: {} does not exist, skipping..".format(artifact))
+      continue 
+
```

Comment:
I was thinking of dropping `os.path.exists()`. It's either a file or a directory or does not exist, right?